ui/pages/import-token/import-token.component.js

```diff
@@ -218,6 +239,20 @@ class ImportToken extends Component {
           customDecimalsError: null,
         });
 
+        break;
+      case standard === 'ERC1155' || standard === 'ERC721':
+        this.setState({
+          collectibleAddressError: this.context.t('collectibleAddressError', [
+            <a
+              className="import-token__collectible-address-error-link"
+              onClick={() => this.props.history.push(ADD_COLLECTIBLE_ROUTE)}
+              key="collectibleAddressError"
+            >
+              {this.context.t('importNFTPage')}
+            </a>,
```

Comment:
Anchor tag without href does not look like good practice to me, @darkwing / @georgewrmarshall  can you plz suggest if it will be better to use a button here.

Reply:
Having trouble getting a `<Button>` to play nice as a passed variable. Do we do this any where else? Can't style it to be in line?
![Screen Shot 2022-01-12 at 2 59 22 PM](https://user-images.githubusercontent.com/34557516/149220627-dbcd9733-d5af-4426-97e4-5cfba550fc29.png)

Reply:
Great discussion! Yes looks like there is a requirement for an inline button component that remains a button and has the style of inline. Currently the [type "link" will change the root html to an `<a>` tag](https://metamask.github.io/metamask-storybook/index.html?path=/docs/ui-components-ui-button-button-stories-js--type-link#type-link) I'll create a ticket. In the mean time you may have to create a custom button component.

Inline button component ticket: #13293

Reply:
https://github.com/MetaMask/metamask-extension/pull/13271/commits/026e802fbb5054f4f8230b6e19b8981a4315be01

Reply:
Actually had to remove the href because it was messing with routing. Not sure the right answer here? Is is really to make a custom `Button` component for now?

Reply:
Lets not block this PR for it. But will be nice to address this in a future PR.